dockerfiles/functional-tests/Dockerfile

```diff
@@ -14,6 +14,9 @@ ENV LANG=en_US.utf8 \
 
 COPY google-chrome.repo /etc/yum.repos.d/google-chrome.repo
 RUN yum update --assumeyes && \
+	curl http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm --output epel.rpm && \
+	yum install --assumeyes epel.rpm && \
```

Comment:
I believe it's enough to do `yum install epel.release` instead of downloading epel.rpm from fedoraproject.

Reply:
:-1: tested on `docker.io/centos:7` image:
```
No package epel.release available.
Error: Nothing to do
```

Reply:
Oops.. misspelling on my side. It's `epel-release`